src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -20,6 +20,7 @@ class LambdaTypeName internal constructor(
     parameters: List<ParameterSpec> = emptyList(),
     val returnType: TypeName = UNIT,
     nullable: Boolean = false,
+    val suspending: Boolean = false,
```

Comment:
Because this isn’t a `private` or `internal` val it’s public API right? I think that’s okay

Reply:
It's API, yes